{"ID": "98337f72-a233-4a57-9bc8-29e6e36932e3", "effective_cve": "CVE-2021-26291", "image": {"exact": "docker.io/sonatype/nexus3@sha256:e8fea6b4279f2b5b24b36170459cb7aa3d6afe999f9d3e3713541be28bae8ec4"}, "label": "FP", "package": {"name": "maven-resolver-api", "version": "1.3.1"}, "timestamp": "2022-11-25T17:26:27+00:00", "tool": "grype@v0.53.1", "user": "westonsteimel", "vulnerability_id": "CVE-2021-26291"}